targets/TARGET_TI/TARGET_CC32XX/TARGET_CC3220SF/gpio_irq_api.c

```diff
@@ -0,0 +1,173 @@
+/* mbed Microcontroller Library
+ * Copyright (c) 2006-2013 ARM Limited
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <stddef.h>
+
+#include "gpio_irq_api.h"
+#include "mbed_error.h"
+#include "cmsis.h"
+#include "PeripheralPins.h"
+
+#include <ti/devices/cc32xx/inc/hw_types.h>
```

Comment:
I believe this should be in "" as the rest of the includes in this file coming within our tree